python/ray/tests/test_output.py

```diff
@@ -337,15 +335,15 @@ def __repr__(self):
         out_str = proc.stdout.read().decode("ascii")
     else:
         out_str = proc.stderr.read().decode("ascii")
-    print(out_str)
 
+    out_str = "".join(out_str.splitlines())
     assert "hi" in out_str, out_str
     assert "(Actor1 pid=" in out_str, out_str
     assert "bye" in out_str, out_str
     assert re.search("Actor2 pid=.*init", out_str), out_str
     assert not re.search("ActorX pid=.*init", out_str), out_str
     assert re.search("ActorX pid=.*bye", out_str), out_str
-    assert not re.search("Actor2 pid=.*bye", out_str), out_str
+    assert re.search("Actor2 pid=.*bye", out_str), out_str
```

Comment:
Hmm interesting, why the change here? cc @rkooo567

Reply:
After removing escape sequences from `out_str`, `re.search("Actor2 pid=.*bye", out_str)` gives `True`. The reason for this, IMO, is that due to presence of `Actor2\r` in `out_str`, `re.search` was giving `False`. However, now, since these escape sequences are removed the above test works fine. Is it something that shouldn't happen?

Reply:
Tests seem to be passing with this change.

Reply:
I think you are right, the test was probably wrong here. Let's see if somebody else has an opinion here and if not let's merge it :)